### ISSUE TITLE
Open in a new window - compact view

### ISSUE THREAD

**Issue by CharacterAJK** [Pitmairen/selection-search#119]
Hey, first of all thank you for the extension - saves time. 

Is there a way to set the extension or change browser settings to make highlighted text to open in a minimalistic type of new window  ?

**Comment by CharacterAJK** [comment]
Yes it would be nice as you stated now it works only in some pages.

**Comment by Pitmairen** [comment]
I have published a new version with a new option to open a search engine in a popup window. The options can be found in the wrench menu next to the search engines. 

It will be available after google has reviewed and approved the release.

**Comment by CharacterAJK** [comment]
Amazing, thank You

**Comment by CharacterAJK** [comment]
Hey, one more thing as you already did that maybe you could help me to understand how to do that or where to find kinda info, how to set chrome to open new window in that type of minimalist appearance. I'm not native English speaker, I don't know how to name the issue to find the answer, as this theme: minimalist pop-up window, it does not help to find what I need :D What I get instead of this type (which you created) window is explanation how to run chrome in app mode or full screen mode :)

**Comment by Pitmairen** [comment]
I think it is called App mode. If you go to `chrome main menu -> more tools -> create shortcut`, it will create a shortcut that will open the web page in app mode.

You can also open app windows using the command line:
```
./chrome --app=https://google.com --new-window
```
where `./chrome` is the path to the chrome executable, which depends on what operating system you are using. The arguments depends on the version of chrome, but the above should work in recent versions.

**Comment by CharacterAJK** [comment]
Thank you
